scripts.d/ta/790_raft_agents.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -ue # Fail with an error code if there's any sub-command/variable error
+
+DESCRIPTION="Examine the number of Raft agents used vs maximum"
+SCRIPT_TYPE="single"
+JIRA_REFERENCE=""
+WTA_REFERENCE=""
+RETURN_CODE=0
+WEKA_VERSION=$(weka version current)
+
+# Check if we can run weka commands
+weka status &> /dev/null
```

Comment:
With `set -ue`, if the user is not logged into weka, the **weka status** command fails and the script terminates abruptly.